test/test_restclient.cc

```diff
@@ -118,6 +121,37 @@ TEST_F(RestClientTest, TestRestClientPOSTBody)
   EXPECT_EQ("restclient-cpp/" RESTCLIENT_VERSION, root["headers"].get("User-Agent", "no url set").asString());
 }
 
+TEST_F(RestClientTest, TestRestClientPostForm)
+{
+  // generating a file name with a timestamp
+  std::ostringstream fileName;
+  time_t rawtime;
+  tm * timeinfo;
+  time(&rawtime);
+  timeinfo = localtime( &rawtime );
```

Comment:
The system time is not a reliable way to ensure you have a unique file name. You should just use [`mkstemp()`](https://www.man7.org/linux/man-pages/man3/mkstemp.3.html) to get a unique filepath instead.

Reply:
Agreed, but it's reliable enough.  If it really bugs you, it can be changed.  Mostly this needs to run on dev systems on and a clean container that's spun up in CI so no worries there.